mobile/src/context/customerContext.interface.ts

```diff
@@ -6,15 +6,20 @@ export type CustomerState = {
   isLoading: boolean;
   state: CustomerStateType | null;
   error: string | null;
+  is_business?: boolean | null | undefined;
```

Comment:
Please follow other conventions and change this variable name from `is_business` to `isBusiness`?